packages/button/stories/button-cta.stories.ts

```diff
@@ -67,6 +67,22 @@ export const withIcon = (): TemplateResult => {
     `;
 };
 
+export const iconSizeOverridden = (): TemplateResult => {
+    return html`
+        <sp-button label="Edit" size="xl" variant=${variant}>
+            <sp-icon slot="icon" size="s">
+                ${HelpIcon({ hidden: true })} Testing
+            </sp-icon>
+        </sp-button>
+        <h1>For testing purposes only</h1>
+        <p>
+            Spectrum specifies that icon elements inside of buttons should share
+            the same "size" rating or have icon accept size values as scoped in
+            the CSS by the button element.
```

Comment:
I'm not totally following what this is trying to say. I'm guessing there's a clearer way to word this.

Reply:
This is a test to ensure that sizing the icon will still work when it's in the scope of a parent element. You shouldn't do this on a regular basis as it is a departure from the Spectrum specification. 

Is that better, or worse, or does it give you any good ideas here?